.github/workflows/nightly.yml

```diff
@@ -0,0 +1,61 @@
+name: Nightly Test run
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 3 * * *'
+
+jobs:
+  integration-tests-standalone:
+    name: Integration Tests (Standalone)
+    runs-on: self-hosted
+    strategy:
+      fail-fast: false
+      matrix:
+        jahia_image: ["jahia/jahia-ee:8", "jahia/jahia-ee-dev:8-SNAPSHOT"]
+    timeout-minutes: 45
+    steps:
+      - uses: jahia/jahia-modules-action/helper@v2
+      - uses: KengoTODA/actions-setup-docker-compose@main
+        with:
+          version: '1.29.2'
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 'lts/*'
+      - uses: actions/checkout@v3
+      - uses: jahia/jahia-modules-action/integration-tests@v2
+        with:
+          module_id: serverSettings
+          jahia_image: ${{ matrix.jahia_image }}
+          testrail_project: Server settings Module
```

Comment:
Can you rename this to: ServerSettings Module

I just created the project in Testrail